factory/22.04/stretch_ros2_humble.repos

```diff
@@ -6,11 +6,11 @@ repositories:
   moveit_visual_tools:
```

Comment:
@hello-atharva Binaries available. Do we need to source build?

Reply:
Ref: [https://github.com/hello-robot/stretch_install/pull/39#discussion_r1207148207](https://github.com/hello-robot/stretch_install/pull/39#discussion_r1207148207)